src/messages.rs

```diff
@@ -940,20 +1016,53 @@ mod tests {
             dst: Authority::ClientManager(name),
             content: part,
         };
-        let hash_msg = unwrap!(routing_message.to_grp_msg_hash());
-        match hash_msg.content {
-            MessageContent::GroupMessageHash(..) => (),
-            _ => panic!("Wrong content for hashed message: {:?}", hash_msg),
-        }
-        assert_eq!(hash_msg, unwrap!(hash_msg.to_grp_msg_hash()));
 
-        let non_hash_routing_msg = RoutingMessage {
-            src: Authority::ClientManager(name),
-            dst: Authority::ClientManager(name),
-            content: MessageContent::GetCloseGroup(MessageId::zero()),
+        // Add a signature which will not correspond to an ID in the first group list.
```

Comment:
Maybe that comment should be a few lines further down, after creating the message.